src/Microsoft.OpenApi.OData.Reader/Edm/EdmModelExtensions.cs

```diff
@@ -124,43 +124,7 @@ public static IEnumerable<IEdmEntityType> FindAllBaseTypes(this IEdmEntityType e
                 current = current.BaseEntityType();
             }
         }
-
-        /// <summary>
-        /// Checks if the <paramref name="baseType"/> is assignable to <paramref name="subtype"/>.
-        /// In other words, if <paramref name="subtype"/> is a subtype of <paramref name="baseType"/> or not.
-        /// </summary>
-        /// <param name="baseType">Type of the base type.</param>
-        /// <param name="subtype">Type of the sub type.</param>
-        /// <returns>true, if the <paramref name="baseType"/> is assignable to <paramref name="subtype"/>. Otherwise returns false.</returns>
-        public static bool IsAssignableFrom(this IEdmEntityType baseType, IEdmEntityType subtype)
```

Comment:
since this method was public, this change is a breaking change.
Would you consider adding the method back and adding an obsolete annotation instead?

Reply:
Oh yeah.